mlos_bench/mlos_bench/tests/launcher_parse_args_test.py

```diff
@@ -116,6 +142,7 @@ def test_launcher_args_parse_1(config_paths: list[str]) -> None:
     cli_args = (
         "--config-paths "
         + " ".join(config_paths)
+        + " --num-trial-runners 5"
```

Comment:
note for a future PR: maybe we can drop those `+` operators now

Reply:
#919